docs/Documentation/UserGuideV0.7.0/2-Concept Key Concepts and Terminology/3-Encoding.md

```diff
@@ -44,6 +44,12 @@ Run-length encoding can also be used to encode floating-point numbers, but it is
 
 GORILLA encoding is more suitable for floating-point sequence with similar values and is not recommended for sequence data with large fluctuations.
 
+* REGULAR
+
+Regular data encoding is more suitable for encoding regular sequence increasing data (e.g. the timeseries with the same time elapsed between each data point), which its performance would be better than TS_2DIFF in this situation.
+
+Regular data encoding method is not suitable for the data with fluctuations (irregular data), which is recommended to use the TS_2DIFF encoder to deal with it.
+
```

Comment:
Simply for your information.
"which .... in this situation" --> "in which case it's better than TS_2DIFF"
"which is recommended to use the TS_2DIFF encoder to deal with it" --> "and TS_2DIFF is recommended to deal with it".

Reply:
Done. Thx